ground/src/main/res/layout/photo_task_frag.xml

```diff
@@ -32,19 +35,19 @@
   <LinearLayout
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
+    android:orientation="vertical"
     android:paddingStart="8dp"
-    android:paddingEnd="8dp"
-    android:orientation="vertical">
+    android:paddingEnd="8dp">
     <com.google.android.material.button.MaterialButton
       android:id="@+id/btn_take_photo"
       style="@style/TaskChipButton.LightGreen"
       android:layout_width="wrap_content"
       android:layout_height="wrap_content"
-      app:icon="@drawable/outline_photo_camera"
-      app:iconTint="?attr/colorOnSurface"
-      android:onClick="@{()-> viewModel.onTakePhotoClick()}"
+      android:onClick="@{()-> fragment.onTakePhoto()}"
       android:text="@string/take_picture"
-      android:visibility="@{viewModel.isPhotoPresent() ? View.GONE : View.VISIBLE}" />
+      android:visibility="@{viewModel.isPhotoPresent() ? View.GONE : View.VISIBLE}"
```

Comment:
We have a binder for this, so you can now use app:visible with `@{viewModel.isPhotoPresent()}`